### PR TITLE
[Android] User Feedback Section Updated

### PR DESCRIPTION
**Pull request by AliRKat**

### CODE REVIEW

**Comment by AliRKat** [comment]
[[SDK, Doc] [Current: Web, Unity] Updating Mumbo Jumbo User Feedback Sections](https://trello.com/c/330rma6c/1886-sdk-doc-current-web-unity-updating-mumbo-jumbo-user-feedback-sections)

**Review by arifBurakDemiray — COMMENTED** [review]
I am not sure here, I will wait for Big D's thoughts

File: android/next.md [hunk 1]
```diff
@@ -1798,13 +1798,15 @@ Countly.sharedInstance().init(config);</code></pre>
...
-  <span style="font-weight: 400;">There are a couple ways of receiving feedback from your users: star-rating dialog, the rating widget and the feedback widgets (survey, nps).</span>
+  <span style="font-weight: 400;">There are several ways to receive user feedback: the Star Rating Dialog and the Feedback Widgets (Survey, NPS, Rating).</span>
 </p>
 <p>
-  <span style="font-weight: 400;">Star-rating dialog allows users to give feedback as a rating from 1 to 5. The rating widget allows users to rate using the same 1 to 5 rating system as well as leave a text comment. Feedback widgets (survey, nps) allow for even more textual feedback from users.</span>
+  <span style="font-weight: 400;">Star Rating Dialog allows users to give feedback by rating it from 1 to 5. Feedback Widgets (Survey, NPS, Rating) allow for even more textual feedback from users.</span>
 </p>
-<h2 id="h_01HAVQDM5VDY31NB5X2KRAKKW9">Ratings</h2>
-<h3 id="h_01HAVQDM5V8DKE0V808A24Z7W0">Star Rating Dialog</h3>
+<div class="callout callout--info">
```
Comment (turtledreams) [inline android/next.md:1806]: lets move this under feedback widget h2

**Review by turtledreams — CHANGES_REQUESTED** [review]
also the picture thingy we talked about

**Review by arifBurakDemiray — APPROVED** [review]

**Pull request merged**